lib/eco_service_web/live/eco_service_live/community_list.ex

```diff
@@ -4,45 +4,73 @@ defmodule EcoServiceWeb.EcoServiceLive.CommunityList do
   alias EcoService.EcoServiceContext
 
   def mount(_params, _session, socket) do
-    params = %{limit: 10, offset: 0}
-    all_communities = EcoServiceContext.fetch_all_communities(params)
+    pag_params = %{limit: 10, offset: 0}
+    all_communities = EcoServiceContext.fetch_all_communities(pag_params)
 
     {:ok,
     socket
     |> assign(:communities, all_communities)
-    |> assign(:limit, params.limit)
-    |> assign(:offset, params.offset)
-    |> assign(:all_records, Enum.count(EcoServiceContext.fetch_all_communities()))
+    |> assign(:limit, pag_params.limit)
+    |> assign(:offset, pag_params.offset)
+    |> assign(:count_of_all_records, Enum.count(EcoServiceContext.fetch_all_communities()))
```

Comment:
I've already given a review for similar code that you've written. Look at the second paragraph of this review comment [here](https://github.com/sangeethailango/Eco-Service/pull/4/files#r1545250779).